pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "dominodatalab-data"
-version = "0.3.1dev2"
+version = "0.3.1dev4"
```

Comment:
are we publishing this to PyPI or already published it? i am not seeing `dev3` yet..

Reply:
dev3 was that Giulio was using in his PR(https://github.com/dominodatalab/domino-data/pull/53/files). didn't want to use the same name. is there anything different i should be doing here?